src/05-derivatives/03-imaging.md

```diff
@@ -0,0 +1,405 @@
+# Imaging data types
+
+This section pertains to imaging data, which characteristically have spatial
+extent and resolution.
+
+## Preprocessed, coregistered and/or resampled volumes
+
+Template:
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        <datatype>/
+            <source_entities>[_space-<space>][_res-<label>][_den-<label>][_desc-<label>]_<suffix>.<ext>
+```
+
+Volumetric preprocessing does not modify the number of dimensions, and so
```

Comment:
The average over time of a four-dimensional NIfTI file would change the number of dimensions - or is this not under consideration here?

Reply:
That's not under consideration for preprocessing. That will come with [functional derivatives](https://bids-specification.readthedocs.io/en/derivatives/05-derivatives/05-functional-derivatives.html).